common/sketch/src/main/java/org/apache/spark/util/sketch/CountMinSketchImpl.java

```diff
@@ -325,27 +321,43 @@ public void writeTo(OutputStream out) throws IOException {
   }
 
   public static CountMinSketchImpl readFrom(InputStream in) throws IOException {
+    CountMinSketchImpl sketch = new CountMinSketchImpl();
+    sketch.readFrom0(in);
+    return sketch;
+  }
+
+  private void readFrom0(InputStream in) throws IOException {
```

Comment:
this is actually a common naming style in java - to have the private version named xxx0